pom.xml

```diff
@@ -20,7 +20,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.fabric8</groupId>
   <artifactId>elasticsearch-cloud-kubernetes</artifactId>
-  <version>2.4.5-SNAPSHOT</version>
+  <version>2.4.4</version>
```

Comment:
No need to change this as the fabric deployment process will update it correctly